torch/fx/experimental/proxy_tensor.py

```diff
@@ -136,9 +143,13 @@ def create_arg(self, a: Any):
             return self.create_node('get_attr', qualname, (), {})
         return super().create_arg(a)
 
+    def trace(self, root, concrete_args):
```

Comment:
Do we need to override this? Can we just push the torch dispatch mode onto the stack within `dispatch_trace`?